src/test/emulators/storage/files.spec.ts

```diff
@@ -173,3 +450,16 @@ describe("files", () => {
     const getPersistenceTmpDir = () => `${tmpdir()}/firebase/storage/blobs`;
   });
 });
+
+/**
+ * get the current date + changeBy SECONDS
+ * @date 7/12/2023 - 4:28:20 PM
+ *
+ * @param {number} changeBy
+ * @returns {string}
+ */
+function getAdjustedDate(changeBy: number): string {
+  const currentDate = new Date().getTime();
+  const adjutedDate = new Date(currentDate + changeBy * SECONDS_TO_MS_FACTOR);
+  return getSignedUrlTimestampFor(adjutedDate);
```

Comment:
This function can be one line:
```
function getAdjustedDate(changeBy: number): string {
  return getSignedUrlTimestampFor(new Date(new Date().getTime() + changeBy * SECONDS_TO_MS_FACTOR));
}
```

Reply:
the only reason I have it like this is for clarity purposes. It is hard to tell what the function is doing in one line